bin/rpm-s3

```diff
@@ -107,15 +107,16 @@ class FileGrabber(object):
 
 
 def getclient(base, region):
+    host_url = "s3.amazonaws.com" if region == "us-east-1" else "s3-{}.amazonaws.com".format(region)
     if os.getenv('AWS_ACCESS_KEY'):
         return boto.connect_s3(
             os.getenv('AWS_ACCESS_KEY'),
```

Comment:
Honestly I'm of the mindset we shouldn't be passing *any* credentials and just use [default credential provider](http://boto3.readthedocs.io/en/latest/guide/configuration.html#configuring-credentials).  That change is already merged to master and isn't as critical as the bug this PR is trying to fix.